inference-engine/src/mkldnn_plugin/mkldnn_plugin.cpp

```diff
@@ -610,5 +620,61 @@ QueryNetworkResult Engine::QueryNetwork(const CNNNetwork& network, const std::ma
     return res;
 }
 
+InferenceEngine::IExecutableNetworkInternal::Ptr Engine::ImportNetwork(
+                                            const std::string &modelFileName,
+                                            const std::map<std::string, std::string> &config) {
+    std::fstream inputStream(modelFileName, std::ios::in | std::ios::binary);
+    if (inputStream.fail()) {
+        IE_THROW() << "Cannot open file to import model: " << modelFileName;
+    }
+    return ImportNetwork(inputStream, config);
```

Comment:
it already has default implementation, can we remove this override?

Reply:
ok, i'll remove it